client/src/app/site/pages/meetings/pages/participants/pages/participant-presence/components/participant-presence/participant-presence.component.ts

```diff
@@ -48,6 +49,7 @@ export class ParticipantPresenceComponent implements OnInit {
         try {
             this.errorMsg = null;
             this.lastChangedUser = this.userRepo.getViewModelByNumber(number)!;
+            this.lastChangedUserPresent = !this.lastChangedUser?.isPresentInMeeting();
             await this.userRepo.togglePresenceByNumber(number);
```

Comment:
Can't you just use the user observable instead of fetching the `ViewUser` and calculate the value based on that? I think I would like that more than an extra variable.

Reply:
I thought about that too but since this is a alert I think it might be better if the content does not change if the presence status gets changed from somewhere else.

Reply:
Ok, I see that. Then, I would remove the `lastChangedUser` variable since it does not seem to be used anywhere else and check for `lastChangedUserPresent === null` to see if the message should be displayed or not.

Reply:
The property `lastChangedUser` is used within the template to display the `full_name`.

Reply:
ok